UnitTest/DataStructuresTests/CircularBufferTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using DataStructures.Lists;
+using Xunit;
+
+namespace UnitTest.DataStructuresTests {
+    public class CircularBufferTest {
+        
+        [Fact]
+        public static void SetsFixedLength() {
+            var circularBuffer = new CircularBuffer<byte>(3);
+            var length = circularBuffer.Length;
+
+            Assert.Equal(3, length);
```

Comment:
Can you add also asserts for properties IsEmpty and IsFilledUp  to be sure it's correctly set?

Reply:
Sure

Reply:
I have added two test cases; CheckIsEmptyProperty and CheckIsFilledProperty, both test cases are GREEN.